src/Hl7.Fhir.Specification/Specification/Terminology/LocalTerminologyService.cs

```diff
@@ -33,89 +33,148 @@ public LocalTerminologyService(IResourceResolver resolver, ValueSetExpanderSetti
 
         internal ValueSet FindValueset(string canonical)
         {
-// Don't want to redo ITerminologyService yet
+            // Don't want to redo ITerminologyService yet
```

Comment:
This was about making the ITerminologyService _async_.  Now that we are adding so many new methods to it....shouldn't anything new we add be async?